docs/docs/add-custom-webpack-config.md

```diff
@@ -9,7 +9,7 @@ es general, te animamos encarecidamente a que contribuyas con tu plugin en la
 Biblioteca de Plugins de Gatsby para que esté disponible para otros 
 (incluyendo a tu futuro yo 😀)._
 
-Para añadir configuraciones de webpack personalizadas, crea (si aun no lo hoy) 
+Para añadir configuraciones de webpack personalizadas, crea (si aun no lo hay) 
```

Comment:
Hola @alvaromontoro, ¿cómo estás? ¿No tendría que ser "aún si no las hay" en vez de "Si aun no lo hay"?

Reply:
@gusma tal y como lo entiendo "if there's not one already" se refiere al fichero `gatsby-node.js` y no a las configuraciones, en cuyo caso la concordancia se haría con singular masculino en lugar de plural femenino. Quizás podría cambiar la traducción y la frase anterior un poco para evitar eso:

> Para añadir una configuración personalizada de webpack, crea (si aún no existe) un fichero `gatsby-node.js` en tu directorio raíz.

¿Se ve mejor?

No relacionado directamente, pero usé _aun_ en lugar de _aún_ porque es como lo vi en otro fichero (no recuerdo cuál) y, aunque prefiero la versión con acento, creo que la RAE lo cambió a sin acento recientemente. Puedo poner el acento si quieres.

Reply:
Si :) Es perfecto. Estaba tan dormido que en realidad lo que quise resaltar fue el typo de Hoy / Hay y no terminé poniendo nada. Gracias!